<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="1" hostname="DESKTOP-OB1OE1J" name="Booking" tests="2" failures="0" timestamp="2021-11-01T11:23:46 IST" time="120.534" errors="1">
  <testcase name="booking" time="0.000" classname="Booking">
    <skipped/>
  </testcase> <!-- booking -->
  <system-out/>
  <testcase name="initialize" time="120.534" classname="Booking">
    <error type="io.appium.java_client.service.local.AppiumServerHasNotBeenStartedLocallyException" message="The local appium server has not been started. The given Node.js executable: C:\Program Files\nodejs\node.exe Arguments: [C:\Users\SURABHIKUMAR\AppData\Roaming\npm\node_modules\appium\build\lib\main.js, --port, 4298, --address, 0.0.0.0, --log, D:\UI Automation\AndroidAutomation_ServiceProvider\Android_OnePlus\Server.log, --session-override] 
Process output: uncaughtException: EPERM: operation not permitted, stat &#039;D:\UI Automation\AndroidAutomation_ServiceProvider\Android_OnePlus\Server.log&#039;
Error: EPERM: operation not permitted, stat &#039;D:\UI Automation\AndroidAutomation_ServiceProvider\Android_OnePlus\Server.log&#039;
C:\Users\SURABHIKUMAR\AppData\Roaming\npm\node_modules\appium\node_modules\winston\lib\winston\logger.js:307
      throw ex;
      ^

NodeError: write after end
    at writeAfterEnd (C:\Users\SURABHIKUMAR\AppData\Roaming\npm\node_modules\appium\node_modules\readable-stream\lib\_stream_writable.js:261:12)
    at PassThrough.Writable.write (C:\Users\SURABHIKUMAR\AppData\Roaming\npm\node_modules\appium\node_modules\readable-stream\lib\_stream_writable.js:305:21)
    at File.log (C:\Users\SURABHIKUMAR\AppData\Roaming\npm\node_modules\appium\node_modules\winston\lib\winston\transports\file.js:185:34)
    at File._write (C:\Users\SURABHIKUMAR\AppData\Roaming\npm\node_modules\appium\node_modules\winston-transport\index.js:103:17)
    at doWrite (C:\Users\SURABHIKUMAR\AppData\Roaming\npm\node_modules\appium\node_modules\winston-transport\node_modules\readable-stream\lib\_stream_writable.js:428:64)
    at writeOrBuffer (C:\Users\SURABHIKUMAR\AppData\Roaming\npm\node_modules\appium\node_modules\winston-transport\node_modules\readable-stream\lib\_stream_writable.js:417:5)
    at File.Writable.write (C:\Users\SURABHIKUMAR\AppData\Roaming\npm\node_modules\appium\node_modules\winston-transport\node_modules\readable-stream\lib\_stream_writable.js:334:11)
    at DerivedLogger.ondata (C:\Users\SURABHIKUMAR\AppData\Roaming\npm\node_modules\appium\node_modules\readable-stream\lib\_stream_readable.js:681:20)
    at DerivedLogger.emit (events.js:388:22)
    at addChunk (C:\Users\SURABHIKUMAR\AppData\Roaming\npm\node_modules\appium\node_modules\readable-stream\lib\_stream_readable.js:298:12)
    at readableAddChunk (C:\Users\SURABHIKUMAR\AppData\Roaming\npm\node_modules\appium\node_modules\readable-stream\lib\_stream_readable.js:280:11)
    at DerivedLogger.Readable.push (C:\Users\SURABHIKUMAR\AppData\Roaming\npm\node_modules\appium\node_modules\readable-stream\lib\_stream_readable.js:241:10)
    at DerivedLogger.Transform.push (C:\Users\SURABHIKUMAR\AppData\Roaming\npm\node_modules\appium\node_modules\readable-stream\lib\_stream_transform.js:139:32)
    at DerivedLogger._transform (C:\Users\SURABHIKUMAR\AppData\Roaming\npm\node_modules\appium\node_modules\winston\lib\winston\logger.js:305:12)
    at DerivedLogger.Transform._read (C:\Users\SURABHIKUMAR\AppData\Roaming\npm\node_modules\appium\node_modules\readable-stream\lib\_stream_transform.js:177:10)
    at DerivedLogger.Transform._write (C:\Users\SURABHIKUMAR\AppData\Roaming\npm\node_modules\appium\node_modules\readable-stream\lib\_stream_transform.js:164:83)
Emitted &#039;error&#039; event on PassThrough instance at:
    at errorOrDestroy (C:\Users\SURABHIKUMAR\AppData\Roaming\npm\node_modules\appium\node_modules\readable-stream\lib\internal\streams\destroy.js:98:101)
    at writeAfterEnd (C:\Users\SURABHIKUMAR\AppData\Roaming\npm\node_modules\appium\node_modules\readable-stream\lib\_stream_writable.js:263:3)
    at PassThrough.Writable.write (C:\Users\SURABHIKUMAR\AppData\Roaming\npm\node_modules\appium\node_modules\readable-stream\lib\_stream_writable.js:305:21)
    [... lines matching original stack trace ...]
    at DerivedLogger.Transform._read (C:\Users\SURABHIKUMAR\AppData\Roaming\npm\node_modules\appium\node_modules\readable-stream\lib\_stream_transform.js:177:10)

">
      <![CDATA[io.appium.java_client.service.local.AppiumServerHasNotBeenStartedLocallyException: The local appium server has not been started. The given Node.js executable: C:\Program Files\nodejs\node.exe Arguments: [C:\Users\SURABHIKUMAR\AppData\Roaming\npm\node_modules\appium\build\lib\main.js, --port, 4298, --address, 0.0.0.0, --log, D:\UI Automation\AndroidAutomation_ServiceProvider\Android_OnePlus\Server.log, --session-override] 
Process output: uncaughtException: EPERM: operation not permitted, stat 'D:\UI Automation\AndroidAutomation_ServiceProvider\Android_OnePlus\Server.log'
Error: EPERM: operation not permitted, stat 'D:\UI Automation\AndroidAutomation_ServiceProvider\Android_OnePlus\Server.log'
C:\Users\SURABHIKUMAR\AppData\Roaming\npm\node_modules\appium\node_modules\winston\lib\winston\logger.js:307
      throw ex;
      ^

NodeError: write after end
    at writeAfterEnd (C:\Users\SURABHIKUMAR\AppData\Roaming\npm\node_modules\appium\node_modules\readable-stream\lib\_stream_writable.js:261:12)
    at PassThrough.Writable.write (C:\Users\SURABHIKUMAR\AppData\Roaming\npm\node_modules\appium\node_modules\readable-stream\lib\_stream_writable.js:305:21)
    at File.log (C:\Users\SURABHIKUMAR\AppData\Roaming\npm\node_modules\appium\node_modules\winston\lib\winston\transports\file.js:185:34)
    at File._write (C:\Users\SURABHIKUMAR\AppData\Roaming\npm\node_modules\appium\node_modules\winston-transport\index.js:103:17)
    at doWrite (C:\Users\SURABHIKUMAR\AppData\Roaming\npm\node_modules\appium\node_modules\winston-transport\node_modules\readable-stream\lib\_stream_writable.js:428:64)
    at writeOrBuffer (C:\Users\SURABHIKUMAR\AppData\Roaming\npm\node_modules\appium\node_modules\winston-transport\node_modules\readable-stream\lib\_stream_writable.js:417:5)
    at File.Writable.write (C:\Users\SURABHIKUMAR\AppData\Roaming\npm\node_modules\appium\node_modules\winston-transport\node_modules\readable-stream\lib\_stream_writable.js:334:11)
    at DerivedLogger.ondata (C:\Users\SURABHIKUMAR\AppData\Roaming\npm\node_modules\appium\node_modules\readable-stream\lib\_stream_readable.js:681:20)
    at DerivedLogger.emit (events.js:388:22)
    at addChunk (C:\Users\SURABHIKUMAR\AppData\Roaming\npm\node_modules\appium\node_modules\readable-stream\lib\_stream_readable.js:298:12)
    at readableAddChunk (C:\Users\SURABHIKUMAR\AppData\Roaming\npm\node_modules\appium\node_modules\readable-stream\lib\_stream_readable.js:280:11)
    at DerivedLogger.Readable.push (C:\Users\SURABHIKUMAR\AppData\Roaming\npm\node_modules\appium\node_modules\readable-stream\lib\_stream_readable.js:241:10)
    at DerivedLogger.Transform.push (C:\Users\SURABHIKUMAR\AppData\Roaming\npm\node_modules\appium\node_modules\readable-stream\lib\_stream_transform.js:139:32)
    at DerivedLogger._transform (C:\Users\SURABHIKUMAR\AppData\Roaming\npm\node_modules\appium\node_modules\winston\lib\winston\logger.js:305:12)
    at DerivedLogger.Transform._read (C:\Users\SURABHIKUMAR\AppData\Roaming\npm\node_modules\appium\node_modules\readable-stream\lib\_stream_transform.js:177:10)
    at DerivedLogger.Transform._write (C:\Users\SURABHIKUMAR\AppData\Roaming\npm\node_modules\appium\node_modules\readable-stream\lib\_stream_transform.js:164:83)
Emitted 'error' event on PassThrough instance at:
    at errorOrDestroy (C:\Users\SURABHIKUMAR\AppData\Roaming\npm\node_modules\appium\node_modules\readable-stream\lib\internal\streams\destroy.js:98:101)
    at writeAfterEnd (C:\Users\SURABHIKUMAR\AppData\Roaming\npm\node_modules\appium\node_modules\readable-stream\lib\_stream_writable.js:263:3)
    at PassThrough.Writable.write (C:\Users\SURABHIKUMAR\AppData\Roaming\npm\node_modules\appium\node_modules\readable-stream\lib\_stream_writable.js:305:21)
    [... lines matching original stack trace ...]
    at DerivedLogger.Transform._read (C:\Users\SURABHIKUMAR\AppData\Roaming\npm\node_modules\appium\node_modules\readable-stream\lib\_stream_transform.js:177:10)


at io.appium.java_client.service.local.AppiumDriverLocalService.start(AppiumDriverLocalService.java:168)
at com.nitara.utils.ServerManager.startServer(ServerManager.java:25)
at com.nitara.GenericBase.GenericBase.initialize(GenericBase.java:60)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.internal.TestInvoker.runConfigMethods(TestInvoker.java:700)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:527)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: org.openqa.selenium.net.UrlChecker$TimeoutException: Timed out waiting for [http://127.0.0.1:4298/wd/hub/status] to be available after 120016 ms
at org.openqa.selenium.net.UrlChecker.waitUntilAvailable(UrlChecker.java:100)
at io.appium.java_client.service.local.AppiumDriverLocalService.ping(AppiumDriverLocalService.java:133)
at io.appium.java_client.service.local.AppiumDriverLocalService.start(AppiumDriverLocalService.java:155)
... 34 more
Caused by: java.util.concurrent.TimeoutException
at java.util.concurrent.FutureTask.get(FutureTask.java:205)
at com.google.common.util.concurrent.SimpleTimeLimiter.callWithTimeout(SimpleTimeLimiter.java:156)
at org.openqa.selenium.net.UrlChecker.waitUntilAvailable(UrlChecker.java:75)
... 36 more
]]>
    </error>
  </testcase> <!-- initialize -->
  <system-out/>
</testsuite> <!-- Booking -->
